test/screenshot/fab/standard.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Fab from '../../../packages/fab';
+
+import '../../../packages/fab/index.scss';
+import './index.scss';
+
+ReactDOM.render((
+    <div>
+      <Fab className="demo-button"><i className="material-icons">favorite</i></Fab>
```

Comment:
These would be more readable to me as:

```
<Fab ..>
  <i ....>whatever</i>
</Fab>
```

What do you think. If so we should add a eslint rule or something